auth/src/main/res/values/styles.xml

```diff
@@ -31,33 +31,6 @@
         <item name="android:fontFamily">sans-serif</item>
     </style>
 
-    <style name="FirebaseUI.Text.T01">
```

Comment:
@samtstern Sure! I learned something new today: you can add the `tools` namespace to your styles.